openforcefield/data/test_forcefields/smirnoff99Frosst.offxml

```diff
@@ -9,7 +9,7 @@
         <!-- https://github.com/openforcefield/smirnoff99Frosst/                                         -->
         <!-- If you absolutely must refer to this file in scholarly work, indicate the file name and     --> 
         <!-- Open Force Field Toolkit version that was used                                              -->
-	<Bonds version="0.3" potential="harmonic" fractional_bondorder_method="None" fractional_bondorder_interpolation="linear">
```

Comment:
Not sure how to get the `ParameterAttribute` behavior to be compatible with existing headers like the old line, but also allow tacking on an XML with the new line

Reply:
Ah, I see what you mean here now. This is tricky. Our `openff-1.2.1.offxml` also specifies `fractional_bondorder_method="None", so can't stack in these params with this either.

Best approach might be to built tests without these incompatible FFs for now? I know that' not a satisfying solution.

Reply:
Whatever the best solution happens to be, I doubt it includes breaking the old files that have been stable for a long time. I'll set up separate files/snippets for testing purposes here, but I think a spec change and corresponding up/downconverter will be inevitable.

Reply:
> The smirnoff99Frosst.offxml is an artifact of a particular point in time, so we should leave it sit.

So, this file isn't actually an important artifact. It's a dangerously-named test file that's been hand-modified and that I get a ton of grief about whenever I try to kill it off. I would be happy to just rename it to `test_forcefields/smirnoff_test.offxml` or something else so that we don't have to worry about people using it in production.

But in the short term, let's just work on getting this running. I'd prefer to follow the pattern of [`ProperTorsions` in this file](https://github.com/openforcefield/openforcefield/blob/9815deeb1546dfbbd1a6a9317f557e7455b10663/openforcefield/data/test_forcefields/smirnoff99Frosst.offxml#L141), and change this to something like:

```
	<Bonds version="0.3" potential="harmonic">
```

and then let the initialized handlers load the default value (`AM1-Wiberg`).